src/python.cc

```diff
@@ -57,7 +49,9 @@
 #include "triton/core/tritonbackend.h"
 #include "triton/core/tritonserver.h"
 
-namespace triton { namespace backend { namespace python {
+namespace triton {
```

Comment:
Can you re-run the formatter? These three should be a single line.

Reply:
I rerun the formatter multiple times. Not sure why it keeps changing these into multiple lines. I can manually put them together.

Reply:
That's odd. Let it be as the formatter changes it. This compiles successfully? Such a formatting usually indicates ill formed "{}" pairs

Reply:
It has been compiling for 2 months.

Reply:
I believe you need .clang-format configuration file in the repo for proper formatting. Look here: https://github.com/triton-inference-server/server/blob/master/.clang-format

Reply:
@tanmayv25 Thanks for pointing this out. After using the .clang-format this issue was resolved!